File: src/backend/NewsTrack.WebApi/Dtos/CommentDto.cs
using System;

namespace NewsTrack.WebApi.Dtos
{
    public class CommentDto
    {
        public Guid Id { get; set; }
        public Guid DraftId { get; set; }
        public Guid? ReplyingTo { get; set; }
        public string CreatedBy { get; set; }
        public DateTime CreatedAt { get; set; }
        public string Content { get; set; }
        public uint Likes { get; set; }
        public uint Replies { get; set; }
    }
}

File: src/frontend/src/app/services/Dtos/CommentDto.ts
export class CommentDto {
    id: string = '';
    draftId: string = '';
    replyingTo: string = '';
    createdBy: string = '';
    createdAt: Date = new Date();
    content: string = '';
    likes: number = 0;
    replies: number = 0;
}